server/manager/src/main/java/org/apache/accumulo/manager/tableOps/delete/CleanUp.java

```diff
@@ -91,18 +92,18 @@ public long isReady(long tid, Manager manager) throws Exception {
     boolean done = true;
     Range tableRange = new KeyExtent(tableId, null, null).toMetaRange();
     Scanner scanner = manager.getContext().createScanner(MetadataTable.NAME, Authorizations.EMPTY);
-    MetaDataTableScanner.configureScanner(scanner, manager);
+    ManagerTabletInfoIterator.configureScanner(scanner, manager);
```

Comment:
This would be a follow on.  As this iterator is intended to find tablets that need work, not sure its suitable for this case of deleting a table.  We could optimize the iterator for this case to avoid looking for things that need splits for example, but not sure if that is worthwhile in terms of code complexity increase.  As long as no harm comes from doing the unnecessary checks, maybe using the ManagerTabletInfoIterator here is fine.